weather_mv/loader_pipeline/netcdf_loader.py

```diff
@@ -309,22 +345,32 @@ def run(argv: t.List[str], save_main_session: bool = True):
         logger.error(f'Unable to create table in BigQuery: {e}')
         raise
 
-    with beam.Pipeline(options=pipeline_options) as p:
-        (
-                p
-                | 'Create' >> beam.Create(all_uris)
-                | 'ExtractRows' >> beam.FlatMap(
-                    extract_rows,
-                    variables=known_args.variables,
-                    area=known_args.area,
-                    import_time=known_args.import_time)
-                | 'WriteToBigQuery' >> WriteToBigQuery(
-                    project=table.project,
-                    dataset=table.dataset_id,
-                    table=table.table_id,
-                    write_disposition=BigQueryDisposition.WRITE_APPEND,
-                    create_disposition=BigQueryDisposition.CREATE_NEVER,
-                    custom_gcs_temp_location=known_args.temp_location)
-        )
-
+    # HACK: Cfgrib assumes the original file will stay around for the entire usage.
+    # TODO: Ideally, we should delete the tmpfile as soon as all usage is finished. However, this would require a
+    #  few changes to the pipeline setup. For now, using a hack to use a tmp dir and clean up after the entire run.
+    with contextlib.ExitStack() as stack:
```

Comment:
what is the value if this if we are only one contextmanager deep? imo, this is the primary value of the exit stack.